src/pages/pressure-those-in-power/index.js

```diff
@@ -0,0 +1,544 @@
+      ]}
+    />
+    <section className={`${ChoiceStyles.banner} ${Styles.banner}`}>
+      <div className={`${GlobalStyles.inner} ${ChoiceStyles.inner}`}>
+        <div
+          className={`${ChoiceStyles.bannerContent} ${Styles.bannerContent}`}
+        >
+          <h1 className={`${ChoiceStyles.bannerTitle} ${Styles.bannerTitle}`}>
+            Pressure those in power to help prevent climate breakdown
+          </h1>
+          <p className={`${ChoiceStyles.bannerDesc} ${Styles.bannerDesc}`}>
+            Climate change is an urgent crisis that cannot be held separate from
+            politics for real change to occur
+          </p>
+
+          <div className={ChoiceStyles.bannerActions}>
+            <a
+              className={ChoiceStyles.bannerCtaPrimary}
+              href="#actions"
+              title="See how you can take action"
+            >
+              <button
+                className={`${ButtonStyles.btnSimple} ${ButtonStyles.btnMd} ${
+                  ButtonStyles.btnBanner
+                } ${ButtonStyles.btnMobile}`}
+              >
+                Learn what you can do
+              </button>
+            </a>
+            <div className={ChoiceStyles.bannerCtaSecondary}>
+              <a href="#how" title="How it happens">
+                Discover how it happens
+              </a>
+            </div>
+          </div>
+        </div>
+        <div className={`${ChoiceStyles.bannerImg} ${Styles.bannerImg}`}>
+          <img
+            src={BannerImg}
+            alt="Yomo"
+            className={`${ChoiceStyles.bannerImgShadow} ${
+              Styles.bannerImgShadow
+            }`}
+          />
+        </div>
+      </div>
+    </section>
+    <section className={`${ChoiceStyles.cba} ${Styles.cba}`}>
+      <div className={`${GlobalStyles.inner} ${ChoiceStyles.cbaInner}`}>
+        Short on time?{" "}
+        <a
+          className={ChoiceStyles.cbaAction}
+          href="#tldr"
+          title="Read a summary"
+        >
+          Get a quick summary
+        </a>
+      </div>
+    </section>
+    <section
+      className={`${ChoiceStyles.how} ${Styles.how}`}
+      id="how"
+      aria-label={"How"}
+    >
+      <div className={`${GlobalStyles.inner} ${ChoiceStyles.howInner}`}>
+        <div className={`${ChoiceStyles.howContent} ${Styles.howContent}`}>
+          <h1 className={`${ChoiceStyles.howTitle} ${Styles.howTitle}`}>
+            How politics, policy and those in power contributes to climate
+            breakdown
+          </h1>
+          <ol
+            className={`${ChoiceStyles.choiceList} ${ChoiceStyles.howList} ${
+              Styles.howList
+            }`}
+          >
+            <li>
+              Those with political power make decisions for the world that
+              affect emissions and the state of our planet.
+            </li>
+            <li>
+              Lobbiest for big oil and other large energy companies use their
```

Comment:
```suggestion
              Lobbists for big oil and other large energy companies use their
```

Reply:
I believe after a quick search it should be lobbyists

Reply:
there was a typo in my suggestion hahaha yes